deepchem/nn/model_ops.py

```diff
@@ -645,6 +645,12 @@ def relu(x, alpha=0., max_value=None):
   return x
 
 
+def selu(x):
```

Comment:
Can you add a link to the https://arxiv.org/pdf/1706.02515.pdf as a docstring?  These number are :ghost: